content/api/power-board.md

```diff
@@ -2,14 +2,16 @@
 title: Power Board
 ---
 
-As there is only one power board, it is not accessed like a list like motors for example:
+The power board can be accessed using the `power_board` property of the `Robot` object.
 
 ```python
 main_board = r.power_board
 ```
 
-## Power Outputs
-The power boards 6 outputs can be turned on and off using the `power_on` and `power_off` functions respectively.
+As there is only one power board, it cannot be accessed from a list like motor boards can.
```

Comment:
Unfortunately, it can. https://github.com/sourcebots/robot-api/blob/master/robot/robot.py#L93

I think a good solution for this would be to remove the `power_boards` property on the robot. However that might lead to weirdness if multiple are plugged in, but teams won't need to worry about that

Reply:
To be honest there's no reason why the kit couldn't support multiple power boards (so long as each one has its own battery), so I would advise against removing `power_boards`.

However, we don't want teams to try and use more than one (at least, not until we've verified it's safe to do so), so I'll remove the reference to `power_boards` from the docs.